test/test_BE.js

```diff
@@ -14,35 +15,7 @@ tape('test for home route ', (t) => {
         t.equal(res.text.substr(0, 15), '<!DOCTYPE html>', 'the respone  should be html file');
         t.equal(res.header['content-type'], 'text/html; charset=utf-8', 'the content-type should equal html');
```

Comment:
You don't need line 16 `t.equal(res.header['content-type'], 'text/html; charset=utf-8', 'the content-type should equal html');`  after using `.expect('Content-Type', /html/)`
so remove line 16